inc/functions/options.php

```diff
@@ -468,9 +468,13 @@ function get_rocket_exclude_defer_js() {
 	];
 
 	if ( get_rocket_option( 'defer_all_js', 0 ) && get_rocket_option( 'defer_all_js_safe', 0 ) ) {
-		$jquery = site_url( $wp_scripts->registered['jquery-core']->src );
+		$jquery            = site_url( $wp_scripts->registered['jquery-core']->src );
```

Comment:
I remember some people messing with `jquery-core` and `jquery-migrate`, and registering their custom jQuery under `jquery`. Do you think it's still a thing today?

Reply:
not often enough to warrant that I think, and often they replaced with the googleapis one so it should be covered by that

Reply:
OK. Let's say that the risk of `$wp_scripts->registered['jquery-core']->src` not being set or being empty can be neglected then.